hack/tools/release/notes/main.go

```diff
@@ -187,6 +193,45 @@ func validateConfig(config *notesCmdConfig) error {
 		}
 	}
 
+	if config.previousReleaseVersion != "" {
+		if err := validatePreviousReleaseVersion(config.previousReleaseVersion); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func validatePreviousReleaseVersion(previousReleaseVersion string) error {
+	// Extract version string from ref format (e.g., "tags/v1.0.0-rc.1" -> "v1.0.0-rc.1")
+	if !strings.Contains(previousReleaseVersion, "/") {
+		return errors.New("--previous-release-version must be in ref format (e.g., tags/v1.0.0-rc.1)")
```

Comment:
```suggestion
		return errors.New("--previous-release-version must be in ref format (e.g. tags/v1.0.0-rc.1)")
```

Same with other error messages.